test/modules/bradc/badUses/useExpr.good

```diff
@@ -1 +1 @@
-useExpr.chpl:4: error: 'use' statements must refer to module or enum symbols (e.g., 'use <module>[.<submodule>]*;')
+useExpr.chpl:4: syntax error: 'use' statements must refer to module or 'enum' symbols
```

Comment:
Hmm, this isn't a syntax error, is it?

Reply:
Nope. I think there's a few other parse errors that shouldn't fall in that category either -- I'll go back over all of them and make any I find normal `ERROR`s.